src/main/java/Board.java

```diff
@@ -9,9 +12,74 @@ public Grid getGrid() {
         return grid;
     }
 
-    public void addMarkToGrid(int position, String mark) {
+    public void takeSquare(int position, String playerMark) {
         int index = position - 1;
-        grid.setSquare(index, mark);
+        grid.setSquare(index, playerMark);
+    }
+
+    public boolean moveValid
```

Comment:
Hi Elle! The indentation is off here. I suggest you to start using linter, it's hard to make 762 lines look perfect without it ;)

Reply:
I will do, thanks!